frontend/js/util.js

```diff
@@ -248,6 +248,18 @@ var util = {
         return _.extend(object, _.object(_.map(properties, function (property) {
             return [property, definition(property)];
         })));
+    },
+
+    /**
+     * Capitalize a string
+     * @param {*} s The string to capitalize
+     * @return {s} The capitalized string
+     */
+    capitalize: function (s) {
+        if (typeof s !== 'string') {
+            return '';
+        }
```

Comment:
If we bother to type check here at all, I would like this to fail louder (i.e. `throw` an `Error`) or be nicer by just returning `s` when it's not a string, so as to not just "swallow" input randomly.

But honestly I wouldn't bother. If people call this with something other than a string, the next line will fail anyway, and they shouldn't do that anyway. :wink: 

```suggestion
```